sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java

```diff
@@ -33,7 +33,7 @@
  */
 public final class CosmosDiagnostics {
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosDiagnostics.class);
-    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
why not private anymore? Are we re-using this anywhere else?

Reply:
Was using in tests to validate.

Reply:
But could revert.